src/frontend/src/style/custom-ace-overrides.css

```
@@ -0,0 +1,70 @@
+.ace-monokai .ace_gutter-active-line {
+  background: #181818 !important;
+  color: #555 !important;
+}
+
+/* Strongest override for Monokai selected line highlight */
+.ace-monokai .ace_marker-layer .ace_active-line,
+.ace-monokai .ace_active-line,
+.ace-monokai .ace_gutter-active-line,
+.ace-monokai .ace_marker-layer .ace_selected-word {
+  background-color: #181818 !important;
+  background: #181818 !important;
```

Biome (1.9.4)

[error] Unexpected shorthand property background after background-color



(lint/suspicious/noShorthandPropertyOverrides)